src/e2e/puppeteer/__tests__/snapshot.ts

```diff
@@ -287,6 +285,30 @@ describe('Font Size: 22', () => {
     await sleep(400)
   })
 
-  // run the snapshot tests at font size 22
+  // // run the snapshot tests at font size 22
   testSuite()
 })
+
+describe('Drag and Drop simalation tests', () => {
+  beforeEach(removeHUD)
+
+  it('Check simulateDrag and simulateDrop effect on DOM', async () => {
+    await simulateDragAndDrop(true, true)
+
+    await paste(`
+      - a
+      - b
+      - c
+      - d
+    `)
+
+    await dragAndDropThought('a', 'd')
+
+    await sleep(1500)
```

Comment:
This seems like a long delay. Can you tell me a little more about why this is needed?

Reply:
I added this so all the ui changes are completely done before taking a snapshot, test was failing sometimes if I did not add this here, maybe we can remove it or decrease the delay, I'll check again if that could work

Reply:
Okay. I know there are some animations that take up to 400ms. I'm not sure what would take 1500ms to update. It would be good to know the reason behind it.

Reply:
I have set the delay in the `dragAndDropThought` file of 500ms which seems to work better for some reason than setting it in the test. I'm not really sure why is this though.